src/Illuminate/Mail/Events/MessageSent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Illuminate\Mail\Events;
+
+use Swift_Message;
+
+class MessageSent
+{
+    /**
+     * The Swift message instance.
+     *
+     * @var \Swift_Message
+     */
+    public $message;
+
+    /**
+     * The Swift transport.
+     *
+     * @var string
+     */
+    public $transport;
+
+    /**
+     * The Swift send result - format depends on the transport.
```

Comment:
but without returning the result of the transport, there is no way to handle, for example, the MessageID on SES to map bounces and other return Mails. 

Because of this I add the transport also, so the Listener can handle the result as it should be.

I think this is an Important part for implementing an most optimized mail-solution for a large scaled project.